lib/Controllers/list.js

```diff
@@ -68,6 +69,14 @@ List.prototype.fetch = function(req, res, context) {
         order.push([sortColumn, 'ASC']);
       }
     });
+    var allowedColumns = this.resource.sort.attributes || Object.keys(model.rawAttributes);
+    var columnNames = order.map(_.first);
+    var disallowedColumns = _.difference(columnNames, allowedColumns);
+    if(disallowedColumns.length) {
+      res.status(400);
+      res.json({error: 'Sorting is not allowed on the attributes: ' + disallowedColumns.join(', ') });
```

Comment:
I know I should probably make more of a standardized way to do this, but when I refactored the validation error stuff we made a loose agreement that status 400 is presumed to be an error, and that error messages would be returned in the form:

{
    "message": "a single error message, or title for multiple errors"
    "errors": [
       "multiple",
       "errors"

    ]
}